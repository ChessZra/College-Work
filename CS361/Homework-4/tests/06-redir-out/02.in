/bin/echo This is a test >file2.out
cat file2.out
file file2.out
/usr/bin/wc file2.out
rm file2.out
file file2.out
