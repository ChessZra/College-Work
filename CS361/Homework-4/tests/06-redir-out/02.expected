cs361sh> cs361sh> This is a test
cs361sh> file2.out: ASCII text
cs361sh>  1  4 15 file2.out
cs361sh> cs361sh> file2.out: cannot open `file2.out' (No such file or directory)
cs361sh> 
